server/graphql/core/src/loader/master_list.rs

```diff
@@ -0,0 +1,76 @@
+use std::collections::HashMap;
```

Comment:
Using a loader to avoid complex joins with master list in stock repo layer 🤔  lots of moving parts